### ISSUE TITLE
invalid images generated by server

### ISSUE THREAD

**Issue by tekumara** [oncilla/old-man-yells-at#20]
When checking pngs saved from the served website:
```
❯ pngcheck old-man-yells-at-fivetran.png
old-man-yells-at-fivetran.png  EOF while reading CRC value
ERROR: old-man-yells-at-fivetran.png
```

pngs generated directly from the command line are fine.

**Comment by oncilla** [comment]
Hi @tekumara 

The code that is executed in the CLI and the website should be the same.
Do you have a sample input that is misbehaving consistently and you are willing to share?

**Comment by tekumara** [comment]
Thanks @oncilla try this jpeg via the [server](https://oncilla.github.io/old-man-yells-at/):

![fivetran](https://github.com/oncilla/old-man-yells-at/assets/125105/8f245c10-d2ae-42dc-9024-6cd9e1346fb4)

**Comment by oncilla** [comment]
Interesting, on my android i get this:

![image](https://github.com/oncilla/old-man-yells-at/assets/5121633/d028f52c-cbd2-4173-8703-d7bf126db1c6)

What platform are you using? 
The website is running webassembly and all the processing happens in your browser.

**Comment by tekumara** [comment]
It displays fine but does not have a valid CRC, ie: if i run pngcheck on the image you shared i get `EOF while reading CRC value`

**Comment by oncilla** [comment]
Hi @tekumara  

This should be fixed by #22. Could you double check if it is working now?

**Comment by tekumara** [comment]
Thanks @oncilla it now works!
